site.config.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+
+const homeReadMe = fs.readFileSync('./README.md', 'utf-8');
```

Comment:
@mjhenkes @bjankord With the current setup, this is the terra-site config file that is pulled in and used to spin up the site.